lib/driver.js

```diff
@@ -30,7 +30,7 @@ let iDevice = function (...args) {
 class IosDriver extends BaseDriver {
   resetIos () {
     this.appExt = ".app";
-    _.defaults(this.opts, {
+    this.serverDetails = {
```

Comment:
We need them again down on line 104/105.

We could remove it if this.opts doesn't actually care about server details?